lisa/util/__init__.py

```diff
@@ -750,3 +753,25 @@ def check_till_timeout(
         sleep(interval)
     if timer.elapsed() >= timeout:
         raise LisaException(f"timeout: {timeout_message}")
+
+
+def retry_with_excluded_exceptions(
+    tries: int,
+    delay: float,
+    exceptions_to_skip: Tuple[Type[Exception], ...] = (),
+) -> Callable[[Callable[..., T]], Callable[..., T]]:
+    def decorator(func: Callable[..., T]) -> Callable[..., T]:
```

Comment:
It can call the retry package internally. If it's a skipped exception, raise it, and retry for others.

Reply:
updated